docs/reference/setup/important-settings/path-settings.asciidoc

```diff
@@ -2,10 +2,14 @@
 [discrete]
 === Path settings
 
+{es} writes the data you index to indices and data streams to a `data`
+directory. {es} writes its own application logs, which contain information about
+cluster health and operations, to a `logs` directory.
+
 For <<targz,macOS `.tar.gz`>>, <<targz,Linux `.tar.gz`>>, and
-<<zip-windows,Windows `.zip`>> installations, {es} writes data and logs to the
-respective `data` and `logs` subdirectories of `$ES_HOME` by default.
-However, files in `$ES_HOME` risk deletion during an upgrade.
+<<zip-windows,Windows `.zip`>> installations, the `data` and `logs` are
+subdirectories of `$ES_HOME` by default. However, files in `$ES_HOME` risk
+deletion during an upgrade.
```

Comment:
Suggest a slight clarification in this sentence, which could be either of these options:
* >the `data` and `logs` directories are subdirectories
* >`data` and `logs` are subdirectories

Reply:
Thanks for catching this @lockewritesdocs.